src/pages/main-page/main-page.tsx

```diff
@@ -0,0 +1,59 @@
+import PlaceCard from '../../components/place-card/place-card';
+import SignIn from '../../components/sign-in/sign-in';
+import Location from '../../components/location/location';
+import Sort from '../../components/sort/sort';
+import Map from '../../components/map/map';
+import { getRandomId } from '../../util';
+
+
+type MainPageProps = {
+  placeCardNumber: number;
+}
+
+export default function MainPage({placeCardNumber}: MainPageProps): JSX.Element {
+  const placeCardId = getRandomId(1, placeCardNumber);
```

Comment:
```suggestion
  const placeCardId = getRandomId(1, placeCardNumber);
```

убрать рандомную генерацию ключей